README.md

```diff
@@ -79,6 +79,7 @@ code will be shared into the container so that the webapp will be dynamically re
 [FIRST RUN]
 $ docker-compose up -d postgres
 $ docker-compose build app
+$ docker-compose build sidekiq
 $ docker-compose run app rake db:setup
```

Comment:
do we need to add `bundle exec` to this line and line 84? I can't get it to work without that.

Reply:
oh good call. will update

Reply:
Added.